ROADMAP.md

```diff
@@ -0,0 +1,135 @@
+# ROADMAP
+
+See [README.md](README.md) for developer getting start instructions.
+
+## Minimal complete pipeline
+
+* `heareval/baseline.py` - Simple unlearned embedding algorithm
+following the API.
+
+* `heareval/tasks/` - One time preprocessing of tasks into train +
+test WAVs and `train/test.csv`.
+
+* `heareval/predict/` - [not implemented] For each task, map from
+API-compatible embeddings to `predicted-test.csv`. (This involves
+a linear model for most task types.)
```

Comment:
Linear model? As in just a one layer projection?

Reply:
@maxsolomonhenry sure why not, we can easily add a hidden layer later. Main point is to try to get something e2e done